src/Features/Core/Portable/FindUsages/SourceReferenceItem.cs

```diff
@@ -35,17 +35,17 @@ internal sealed class SourceReferenceItem
         public bool IsWrittenTo { get; }
 
         /// <summary>
-        /// Additional information about the reference.
+        /// Additional properties for the reference that can have multiple values.
         /// Each entry represents a key-values pair of data. For example, consider the below entry:
         ///     { "ValueUsageInfo" } = { "Read", "Write" }
         /// This entry indicates that the reference has additional value usage information which indicate
         /// it is a read/write reference, such as say 'a++'.
         /// </summary>
-        public ReferenceUsageInfoMap ReferenceUsageInfo { get; }
+        public ReferenceUsageInfoMap AdditionalPropertiesWithMultipleValues { get; }
```

Comment:
Should eventually rename `ReferenceUsageInfoMap` to `AdditionalPropertiesWithMultipleValuesMap`